src/app/clusters/group-key-mgmt-server/group-key-mgmt-server.cpp

```diff
@@ -401,7 +401,32 @@ ValidateKeySetWriteArguments(const chip::app::Clusters::GroupKeyManagement::Comm
     return Status::Success;
 }
 
-constexpr uint16_t GroupKeyManagementAttributeAccess::kClusterRevision;
+bool ValidateAndGetProviderAndFabric(chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
+                                     Credentials::GroupDataProvider ** outGroupDataProvider, const FabricInfo ** outFabricInfo)
+{
+    VerifyOrDie((outGroupDataProvider != nullptr) && (outFabricInfo != nullptr));
```

Comment:
should we validate commandObj as well since we are using it below? Or would we segfault anyway if null?

Reply:
Maybe we should just return false on invalid arguments instead of die.

Reply:
Added VerifyOrDie for commandObj as well.